osm.py

```diff
@@ -106,7 +106,11 @@
 
         { "copy": "snippets" },
         { "copy": "themes" }
-    ]
+    ],
+    "backup": {
+        "archive_format_priority_list": ["zip", "gztar", "bztar", "xztar", "tar"],
```

Comment:
@peterkaminski the idea here is that since any one format might not be available, better to have a list of preferred formats, and work left to right until `make_archive` works.
I'm open to better names for the keys/values here.

Reply:
looks fine to me.